### DISCUSSION TITLE
MikoPbx phone app

### DISCUSSION THREAD

**Discussion by bnamaz** [mikopbx/Core#311] [General ( Обо всём )]
Hello, 
I installed MikoPbx. 
Now I want to install the app client phone on my android phone. Can you assist me, please?

**Comment by boffart** [comment]
Use zoiper
https://help.freshphone.co.za/support/discussions/topics/17000004999

**Reply by bnamaz** [reply]
Can I  choose program another from zoiper?

**Comment by boffart** [comment]
any SIP phone convenient for you will do
I use **Groundwire**

**Reply by bnamaz** [reply]
Thanks @boffart  i will try

**Reply by ogt111** [reply]
For Android, I use Linphone

https://play.google.com/store/apps/details?id=org.linphone

My colleague use Sipnetic

https://play.google.com/store/apps/details?id=com.sipnetic.app

You can use any SIP phone that you find at GooglePlay

**Comment by batboyboy1990** [comment]
portSIP ! and Default Accoun intenetcall on almos android phone!
